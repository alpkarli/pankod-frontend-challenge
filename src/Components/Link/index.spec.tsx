import { create } from "react-test-renderer";
import { LinkWithComponent } from "@Components";

describe("LinkWithComponent", () => {
    it("should match snapshot", () => {
        const component = create(
        <LinkWithComponent to="/movies">
            Children
        </LinkWithComponent>
        );
        expect(component.toJSON()).toMatchSnapshot();
    })
});