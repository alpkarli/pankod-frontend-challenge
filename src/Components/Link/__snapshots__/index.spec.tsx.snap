// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkWithComponent should match snapshot 1`] = `
<a
  href="/movies"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  Children
</a>
`;
